### PR TITLE
Update svd_schema.txt

### PR DESCRIPTION
**Pull request by jkrech**
Fixed description for choices of bit-field specifications, where child elements did not match parent.
Responding to: https://github.com/ARM-software/CMSIS_5/issues/1646

### CODE REVIEW

File: doxygen/src/svd_schema.txt [hunk 1]
```diff
@@ -1824,7 +1824,7 @@ A field may define an \em enumeratedValue in order to make the display more intu
         <td>1..1</td>
     </tr>
     <tr  class="choice">
-        <td colspan="4"><em>1. bitRangeLsbMsbStyle</em> </td>
+        <td colspan="4"><em>2. bitRangeOffsetWidthStyle</em></td>
```
Comment (JonatanAntoni) [inline doxygen/src/svd_schema.txt:1827]: This leads to missordering in the generated docs:

```suggestion
        <td colspan="4"><em>1. bitRangeOffsetWidthStyle</em></td>
```

**Review by JonatanAntoni — CHANGES_REQUESTED** [review]
I think we should keep the order (1, 2, 3) in the table. Having (2, 1, 3) looks a bit funny.

**Comment by jkrech** [comment]
Well spotted. Thanks.

**Comment by AndreySmirnov81** [comment]
Maybe it's better to keep the same order as in SVD Schema?

https://github.com/Open-CMSIS-Pack/svd-spec/blob/2567db1a77f12f30776ca986cda8448897a78242/schema/CMSIS-SVD.xsd#L452-L459

**Comment by JonatanAntoni** [comment]
@jkrech, is this still relevant? Shall we update the PR and get it merged?

**Review by JonatanAntoni — APPROVED** [review]

**Pull request merged**